src/Symfony/Component/DependencyInjection/Tests/Attribute/AutowireInlineTest.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Tests\Attribute;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\DependencyInjection\Attribute\AutowireInline;
+use Symfony\Component\DependencyInjection\Exception\LogicException;
+use Symfony\Component\DependencyInjection\Reference;
+
+class AutowireInlineTest //extends TestCase
```

Comment:
@DaDeather I disabled your test case because it tested behaviors that I removed. Validation is now delegated to existing code infrastructure in YamlFileLoader (similarly to what we do in `RegisterAutoconfigureAttributesPass`)
Would you be able to update it to make it fit the new implem?

Reply:
I adapted the changes you made in the tests here:
https://github.com/symfony/symfony/pull/52820/commits/8215026333c5071e9be4efc4792e49e90cef1eb7